SPECS/golang/golang-1.16.spec

```diff
@@ -13,7 +13,7 @@
 Summary:        Go
 Name:           golang
 Version:        1.16.15
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Don't we need to increment the `Release` tag for any packages depending on Golang now as well? I think they mostly link statically, so a rebuild might be in order.

Reply:
Super helpful, Thank you Pawel. Updated dependency SPECs.